Examples/Example-SwiftUI/Example-SwiftUI/Configuration.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Configuration.swift
+//
+//  Created by Mathew Gacy on 12/18/22.
+//  Copyright © 2022 Blue Triangle. All rights reserved.
+//
+
+import Foundation
+
+private enum Configuration {
+    enum Key: String {
+        case baseURL = "_BASE_URL"
+        case siteID = "_SITE_ID"
+    }
+
+    static func value<T>(for key: Key) -> T where T: LosslessStringConvertible {
```

Comment:
It is not clear to me the value of implementing this as generic since all values will be concrete instances of strings.

Reply:
Good point